docs/_posts/2018-08-10-fretboard.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: post
+title: "Fretboard: A/B Testing Framework for Android"
```

Comment:
Let's add GSoC to the title:

"☀️ Google Summer of Code: Fretboard - A/B Testing Framework for Android"

Reply:
Yes, I forgot about that. Now it's added :)